### PR TITLE
Change import from swatisehgal/topologyapi to noderesourcetopology-api

### PR DESCRIPTION
**Pull request by swatisehgal**
This patch ensures that NFD uses the latest implementation of [noderesourcetopology API](https://github.com/k8stopologyawareschedwg/noderesourcetopology-api).

### CODE REVIEW

File: pkg/topologyupdater/topology-updater.proto [hunk 1]
```diff
@@ -26,22 +26,35 @@ service NodeTopology{
...
+    string parent = 3;
+    repeated CostInfo costs = 4;
+    repeated AttributeInfo attributes = 5;
+    repeated ResourceInfo resources = 6;
 
 }
+
+message CostInfo {
+    string name = 1;
+	int32 value = 2 ;
```
Comment (fromanirh) [inline pkg/topologyupdater/topology-updater.proto:45]: nit: weird spacing

File: pkg/topologyupdater/topology-updater.proto [hunk 2]
```diff
@@ -26,22 +26,35 @@ service NodeTopology{
...
+}
+
+message AttributeInfo {
+    string name = 1;
+	int32 value = 2 ;
```
Comment (fromanirh) [inline pkg/topologyupdater/topology-updater.proto:50]: nit: weird spacing

File: pkg/resourcemonitor/noderesourcesaggregator.go [hunk 3]
```diff
@@ -98,30 +100,32 @@ func (noderesourceData *nodeResources) Aggregate(podResData []PodResources) map[
 		}
 	}
 
-	zones := make(map[string]*topologyv1alpha1.Zone)
-
+	// zones := make(topologyv1alpha1.ZoneList, 0)
```
Comment (fromanirh) [inline pkg/resourcemonitor/noderesourcesaggregator.go:103]: we can remove this, right?

**Review by fromanirh — COMMENTED** [review]
initial review

File: pkg/resourcemonitor/noderesourcesaggregator.go (hunk 3 above)
Comment (swatisehgal) [reply pkg/resourcemonitor/noderesourcesaggregator.go]: Yes, removed.

**Review by fromanirh — COMMENTED** [review]
/approve
/lgtm
I was surprised by the change about `intstr`, which seems inherited by k8s, so we don't really have a choice here. Everything else is straightforward.

**Pull request merged**